specs/ceph_installer_wrapper.adoc

```diff
@@ -54,8 +54,45 @@ Tendrl API provides an interface to create a Ceph cluster.
 Sample JSON
 
 ----------
-POST '/api/v1/CreateCluster'
-
+POST '/api/1.0/CreateCluster'
+
+{
+  "sds_type": "ceph",
+  "sds_version": "10.2.5",
+  "name": "MyCluster",
+  "node_configuration": {
+    "3a95fd96-876d-439a-a64d-70332c069aaa": {
+      "role": "ceph/mon"
```

Comment:
For mon node, configure call needs details like `fsid`, `name`, `cluster_network`, `public_network`, `interface`. I feel these fields also should be passed here while create cluster flow either as part of top level fields or mon-node details.

Generated `fsid` for cluster could be top level, whereas `interface`, `cluster_network` and `public_network` are specific to node, so should be passed as part of mon-node. The generated `fsid` should be written at `/etc/tendrl/{integration name}/integration_id` once cluster creation is successful.